lib/debug/tracer.rb

```diff
@@ -58,14 +55,18 @@ def skip? tp
       if tp.path.start_with?(__dir__) ||
          tp.path.start_with?('<internal:') ||
          ThreadClient.current.management? ||
-         (@pattern && !tp.path.match?(@pattern) && !tp.method_id&.match?(@pattern)) ||
+         skip_with_pattern?(tp) ||
```

Comment:
It seems to remove `!tp.method_id&.match...`. Does it intentional?

Reply:
From my understanding, only `CallTracer`'s tp event would have `method_id`. So I extended this behavior in `CallTracer#skip_with_pattern?`